content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Bundle Index Image"
+weight: 3
+description: >
+    Bundle index image example and references
+---
+
+A bundle is an operator packaging construct which contains an operator definition and manifests which ultimately determine how the operator is deployed onto a Kubernetes cluster. Bundles are the preferred mechanism within OLM going forward to package and deploy Operators. Operator developers are encouraged to use the Bundle format over PackageManifests.
+
+To build an Operator index image, we must first have an Operator that is ready to be deployed. To create an Operator, we can use `operator-sdk` to scaffold and construct your Operator. 
+
+Once your Operator is built, the code is built into a container image. If you use [Operator SDK](https://sdk.operatorframework.io/docs/) to build your Operator, it contains a `Makefile` that you can use to build and push your Operator image. To create and push your Operator's image to a registry, use the following command:
+
+```
+make docker-build docker-push IMG=<registry>/<username>/sample-operator:vX.X.X
+```
+
+### Building the Operator Bundle Image
+
+For an Operator to be deployed it has to be bundled. We can create the Bundle using one of the following command:
+```
+make bundle IMG=<registry>/<username>/sample-operator:vX.X.X
+```
+
+The `make bundle` command also creates a Dockerfile, named bundle.Dockerfile, which is used to build a bundle image. The bundle image is an [OCI](https://github.com/opencontainers/image-spec) image that holds the generated on-disk bundle manifest and metadata files.
+
+### Building the Operator Index Image
```

Comment:
The topics in the rest of the document sound like `Tasks`, why do we want to include them under `Concepts`?